content/community/contributing/index.md

```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
 icon: tasks
 ---
  
-
-We are welcoming new contributors to the Keptn project!
+We welcome new contributors to the Keptn project,
+regardless of your background and experience!
 
 ## Getting started
 
+1. Join us on [Keptn Slack](/community/#slack).
+2. Join the `#help-contributing` channel.
+   It can be used for any questions related to Keptn contributions.
+   See the links to more specific channels, e.g. working groups, below.
+3. Explore the newcomer friendly issues, see the links below.
+
+## Keptn development
+
+If you are interested 
```

Comment:
Is something missing here?  Or perhaps this phrase is vestigal and should be deleted?